plugins.xml

```diff
@@ -14021,4 +14021,35 @@ the registration of DOIs with mEDRA.</p>]]></description>
 			<description locale="es_ES">Nueva funcionalidad: Actualización de datos preservados en la Red Cariniana. El envío de datos para preservación ahora envía solo el XML actualizado después del primer envío. Las actualizaciones automáticas a través de tareas programadas monitorean diferencias y envían correos electrónicos de actualización a la Red Cariniana. Las tareas programadas se activan automáticamente por el plugin Acron.</description>
 		</release>
 	</plugin>
+	<plugin category="blocks" product="navigation">
+		<name locale="en">Navigation Block Plugin</name>
+		<name locale="en_US">Navigation Block Plugin</name>
+		<homepage>https://github.com/pkp/tutorialExample</homepage>
+		<summary locale="en">A sidebar block plugin that allows the user to navigate by articles and issues.</summary>
+		<summary locale="en_US">A sidebar block plugin that allows the user to navigate by articles and issues.</summary>
+		<description locale="en"><![CDATA[<p>This plugin creates an additional sidebar block. When added to a sidebar, it displays navigation tools for browsing issues and articles.</p>]]></description>
+		<description locale="en_US"><![CDATA[<p>This plugin creates an additional sidebar block. When added to a sidebar, it displays navigation tools for browsing issues and articles.</p>]]></description>
+		<maintainer>
+			<name>Martin Brändle</name>
+			<institution>Swiss Chemical Society</institution>
+			<email>web.editor@chimia.ch</email>
+		</maintainer>
+		<release date="2025-08-04" version="1.0.0.0" md5="85b86c9b888a7962319ef7f7ba4ca90f">
+			<package>https://github.com/mpbraendle/navigation/releases/download/v1.0.0.0/navigation-block-ojs34-v1.0.0.0.tar.gz</package>
+			<compatibility application="ojs2">
```

Comment:
If this is compatible with all 3.4 releases, then you can just use:

`<version>~3.4.0.0</version>`

Reply:
Ups. Thanks, I will update.